lib/membrane_element_rtp_aac/payloader.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Membrane.RTP.AAC.Payloader do
+  @moduledoc "TODO"
+
+  use Membrane.Filter
+
+  alias Membrane.Buffer
+  alias Membrane.{AAC, RTP}
+  alias Membrane.RTP.AAC.Utils
+
+  def_input_pad :input, accepted_format: %AAC{encapsulation: :none}
+  def_output_pad :output, accepted_format: %RTP{}
+
+  def_options bitrate: [
```

Comment:
In case we decide to keep that option, let's be persistent and let's call it `mode` (as RFC names it and as we did in the depayloader)